### DISCUSSION TITLE
issue in dompdf.php file

### DISCUSSION THREAD

**Discussion by santoshkalukunda** [dompdf/dompdf#2692] [Q&A]
issue in line number 605
![image](https://user-images.githubusercontent.com/38316834/145600969-77c2ca06-050d-4c39-8a76-eb209cbddf05.png)

**Comment by bsweeney** [comment]
A little more detail would be useful, but looking at that particular line you are correct. We're calling the getAttribute method, which is part of the DOMElement class, not the DOMNode class. It doesn't throw an error in this case because the node we're accessing is actually a DOMElement. But the specific logic we use can be better written.

**Reply by bsweeney** [reply]
see #2693
